erpnext/docs/user/manual/en/accounts/journal-entry.md

```diff
@@ -68,4 +68,38 @@ of each year reducing its value by a certain percentage.
 > Note: There may be regulations in your country that define by how much
 amount you can depreciate a class of Assets.
 
+#### Credit Note
+
+Credit note is a note which is given to the customer when we get his returned 
+goods. By giving this note to customer, we try to tell that we are crediting his 
+account with his returned goods amount.
+
+  * Debit: Sales Return Account
+  * Credit: Customer Account
+  
+#### Debit Note
+
+Debit note is a note which is given to the supplier or accepted as a credit note 
+from our supplier when we return goods. By giving this note, we try to tell that 
+we are debiting his account with returned goods amount. 
+
+  * Debit: Supplier Account
+  * Credit: Purchase Return Account 
+
+#### Excise Entry
```

Comment:
This is only for India, maybe this should be a separate article (maybe we should have separate sections for country-wise pages)